src/tests/ftest/server/daos_server_restart.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+from apricot import TestWithServers
+from server_utils import ServerFailed
+from daos_utils import DaosCommand
+
+
+class DaosServerTest(TestWithServers):
+    """Daos server tests.
+
+    Test Class Description:
+        Test to verify that the daos_io_server, daos_server starts/stops
+        and reformat.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a DaosServerTest object."""
+        super(DaosServerTest, self).__init__(*args, **kwargs)
+
+    def restart_daos_server(self):
+        """method to perform server stop and start"""
+
+        try:
+            self.server_managers[0].stop()
+        except ServerFailed as error:
+            self.fail(
+                "##Failed stoping server, {}".format(error))
+        try:
+            self.server_managers[0].start()
+        except ServerFailed as error:
+            self.fail(
+                "##Failed restartng server, {}".format(error))
+
+    def restart_daos_io_server(self, force=False):
+        """method to perform io_server stop and start by dmg"""
+
+        try:
+            self.server_managers[0].dmg.system_stop(force)
+        except CommandFailure as error:
+            self.fail(
+                "##Failed stoping daos io-server by dmg, {}".format(error))
+        try:
+            self.server_managers[0].dmg.system_start()
+        except CommandFailure as error:
+            self.fail(
+                "##Failed starting daos io-server by dmg, {}".format(error))
+
+    def get_pool_list(self):
+        """method to get the pool list contents"""
+        pool_list = self.get_dmg_command().get_output("pool_list")
+        self.log.info(
+            "get_pool-list: {}".format(pool_list))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments